addons/airway/functions/fnc_treatmentAdvanced_RemoveAirwayItemLocal.sqf

```diff
@@ -0,0 +1,43 @@
+#include "script_component.hpp"
+/*
+ * Author: Miss Heda
+ * Removing Guedeltubus
+ *
+ * Arguments:
+ * 0: Medic <OBJECT>
+ * 1: Patient <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [player, cursorTarget] call kat_airway_fnc_treatmentAdvanced_RemoveGuedeltubusLocal;
+ *
+ * Public: No
+ */
+
+params ["_medic", "_patient", "_item"];
+
+_patient setVariable [QGVAR(airway_item), "", true];
+_patient setVariable [QGVAR(airway), false, true];
+_patient call FUNC(handleAirway);
+
+if (_item isEqualTo "Larynxtubus") then {
+    _patient call FUNC(handlePuking);
+};
+
+if !(GVAR(ReusableAirwayItems)) exitwith {};
+    
+if (_item isEqualTo "Larynxtubus") then {
+    if (_medic canAdd "kat_larynx") then {
+        _medic addItem "kat_larynx";
+    } else {
```

Comment:
what if patient doesnt have enough space?
imo it should be done with ACEFUNC(common,addToInventory - look on my PR for reference, at least item is not lost but dropped (look at #301  for reference